packages/gatsby-theme-carbon/src/components/PageTabs/PageTabs.js

```diff
@@ -15,12 +15,15 @@ import {
 export default class PageTabs extends React.Component {
   render() {
     const { tabs, slug } = this.props;
-    const currentTab = slug.split('/').filter(Boolean).slice(-1)[0];
+    const newSlug = slug.replace(/[^a-zA-Z0-9-\/]/g, ''); // removes any characters that aren't letters, numbers or dash/fwd slash
```

Comment:
Do we still need this? If so, we could use slugify w/ the same settings to remove the stuff so we can guarantee we're comparing apples to apples.

Reply:
So slugify the slug? This part is for the initial landing tab, the line where we slugify the tab name is just for when we're looking for what to replace when switching tabs, so we still need something to replace the characters on the initial tab. Are you suggesting something like:
`const newSlug = slugify(slug, { lower: true, strict: true });`

Reply:
^ hope that makes sense regarding the slug v tab regex @vpicone

Reply:
@jnm2377 the slug comes from `location.pathname` in the Default template so it should already be a valid URL. After the update to use strict slugify below, I'm not sure what links they would click that could take them to a URL with special characters in it.